### PR TITLE
Add publish block to gradle.build

### PR DESCRIPTION
**Pull request by Mike-Velarde**
added the publish block to gradle.build to enable publishToMavenLocal task

### CODE REVIEW

**Review by klaemann — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -46,19 +47,28 @@ repositories {
...
-        url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = group
+            artifactId = artifact
+            version = version
+
+            from components.java
+
```
Comment (ford-arnett-br) [inline build.gradle:58]: Okay so apparently that was the minimum amount of code needed. Interesting

File: build.gradle [hunk 2]
```diff
@@ -46,19 +47,28 @@ repositories {
...
-
-        def baseUrl = "file://Volumes/Builds/qa/automation/maven-repository"
-        def releasesRepoUrl = "$baseUrl/releases"
-        def snapshotsRepoUrl = "$baseUrl/snapshots"
-        url = version.endsWith('SNAPSHOT') ? snapshotsRepoUrl : releasesRepoUrl
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = group
+            artifactId = artifact
```
Comment (ford-arnett-br) [inline build.gradle:54]: I am not a gradle expert by any means, but I'm curious why this is a variable above and then only used here?

**Review by ford-arnett-br — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (Mike-Velarde) [reply build.gradle:54]: just to keep it close to the others, all three of these variables are needed for publishing to your local .m2 so keeping them together just made sense to me

Comment (Mike-Velarde) [reply build.gradle:54]: im a long way from gradle expert also so I just made minimum changes

Comment (ford-arnett-br) [reply build.gradle:54]: the def before artifact above makes it different so that's why I was asking

**Pull request merged**